src/Input/Silk.NET.Input.Common/Internals/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,9 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Silk.NET.Input.Sdl")]
+[assembly: InternalsVisibleTo("Silk.NET.Input.Glfw")]
```

Comment:
Specify in csproj see <https://github.com/Ultz/Silk.NET/blob/7675fc3cc7d36f9b582ee2c4ad7594f58d8653bb/src/Maths/Silk.NET.Maths/Silk.NET.Maths.csproj#L28-L30>